specifications/oas/dcsaorg-DCSA_EEC-0.11-alpha-resolved.yaml

```diff
@@ -35,7 +35,7 @@ paths:
           type: string
           example: "1"
       requestBody:
-        description: the transferblock containing the B/L document along with the endorsement chain.
+        description: the transferblock containing the B/L document along with the endorcement chain.
```

Comment:
the `c` spelling is new for me.  Are you confident in that one?

Reply:
no this is indeed wrong. I fixed the spelling mistake